pkg/manifest/environment_test.go

```diff
@@ -38,21 +40,23 @@ func TestEnv(t *testing.T) {
 }
 
 func fakeGitopsDir(t *testing.T) (string, func()) {
-	tmpDir, cleanUp := makeTempDir(t)
+	appFS := afero.NewOsFs()
```

Comment:
`NewOsFs()` still refers to the underlying filesystem. Unit tests can utilize `NewMemMapFs()` which uses an in-memory mock file system.